packages/react-magma-dom/src/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,322 @@
+  span {
+    display: flex;
+    text-align: left;
+  }
+  svg{
+    height:0;
+  }
+  div > span{
+    height: auto;
+  }
+  div div {
+    margin:3px 0;
+    span{
+      margin:0
+    }
+  }
+  div div span:last-child{
+    margin: 4px 0 0 0;
+  } 
+`;
+
+// Stepper!
+export const Stepper = React.forwardRef<HTMLDivElement, StepperProps>(
+  (props, ref) => {
+    const {
+      breakpoint,
+      breakpointLayout,
+      children,
+      currentStep,
+      layout = StepperLayout.showLabels,
+      stepLabel,
+      isInverse: isInverseProp,
+      testId,
+      ...rest
+    } = props;
+    const theme = React.useContext(ThemeContext);
+    const i18n = React.useContext(I18nContext);
+    const isInverse = useIsInverse(isInverseProp);
+
+    // Controls the varying layouts with a set breakpoint number between a breakpointLayout type and / or a layout type.
+
+    const [showLabelsLayout, setShowLabelsLayout] = React.useState(false);
+    const [hideLabelsLayout, setHideLabelsLayout] = React.useState(false);
+    const [summaryViewLayout, setSummaryViewLayout] = React.useState(false);
+
+    React.useEffect(() => {
+      setHideLabelsLayout(layout === StepperLayout.hideLabels);
+      setShowLabelsLayout(layout === StepperLayout.showLabels);
+      setSummaryViewLayout(layout === StepperLayout.summaryView);
+    }, [layout]);
+
+    React.useEffect(() => {
+      setHideLabelsLayout(breakpointLayout === StepperLayout.hideLabels);
+      setShowLabelsLayout(breakpointLayout === StepperLayout.showLabels);
+      setSummaryViewLayout(breakpointLayout === StepperLayout.summaryView);
+    }, [breakpointLayout]);
+
+    const [windowWidth, setWindowWidth] = React.useState(window.innerWidth);
+
+    React.useEffect(() => {
+      function handleResize() {
+        setWindowWidth(window.innerWidth);
+        if (window.innerWidth < breakpoint && breakpoint) {
+          setShowLabelsLayout(breakpointLayout === StepperLayout.showLabels);
+          setHideLabelsLayout(breakpointLayout === StepperLayout.hideLabels);
+          setSummaryViewLayout(breakpointLayout === StepperLayout.summaryView);
+        }
+
+        if (window.innerWidth > breakpoint || !breakpoint) {
+          setShowLabelsLayout(layout === StepperLayout.showLabels);
+          setHideLabelsLayout(layout === StepperLayout.hideLabels);
+          setSummaryViewLayout(layout === StepperLayout.summaryView);
+        }
+        if (showLabelsLayout) {
+          setHideLabelsLayout(false);
+          setSummaryViewLayout(false);
+        } else if (hideLabelsLayout) {
+          setShowLabelsLayout(false);
+          setSummaryViewLayout(false);
+        } else if (summaryViewLayout) {
+          setShowLabelsLayout(false);
+          setHideLabelsLayout(false);
+        }
+      }
+
+      window.addEventListener('resize', handleResize);
+      handleResize();
+
+      return () => window.removeEventListener('resize', handleResize);
+    }, [windowWidth]);
+
+    // Step states
```

Comment:
I don't think this comment is still applicable here? Or maybe it's above the wrong line of code